plio/models.py

```diff
@@ -6,6 +6,20 @@
 from plio.config import plio_status_choices, item_type_choices, question_type_choices
 
 
+class Image(SafeDeleteModel):
+    _safedelte_policy = SOFT_DELETE
+
+    image_url = models.ImageField("Image", upload_to="images")
```

Comment:
```suggestion
    url = models.ImageField("Image", upload_to="images")
```

Reply:
I think we also need to add alt text - all platforms have it and it's an important element of accessibility.

Reply:
we can set some default alt text and might not need to provide option to modify it - but at least something should be there.

Reply:
What is alt text?

Reply:
https://www.w3schools.com/tags/att_img_alt.asp